net-p2p/qbittorrent/qbittorrent-3.3.4.recipe

```diff
@@ -52,7 +57,7 @@ BUILD_PREREQUIRES="
 BUILD()
 {
 	bootstrap.sh
-	BOOST_CPPFLAGS=-I/system/$relativeHearersDir runConfigure \
+	BOOST_CPPFLAGS=-I/system/$relativeHeadersDir runConfigure \
```

Comment:
I wonder how did it work before? Maybe passing BOOST_CPPFLAGS is not needed anymore?